frontend/packages/ceph-storage-plugin/src/components/kms-config/vault-config.tsx

```diff
@@ -64,6 +64,7 @@ export const ValutConfigure: React.FC<KMSConfigureProps> = ({
   };
 
   const setAuthMethod = (authMethod: VaultAuthMethods) => {
+    vaultStateClone.authValue.value = '';
```

Comment:
> setAuthValue('')

Reply:
Just intializing  vaultStateClone.authValue.value = ''; wont work, value needs to be validated tusing vaultStateClone.authValue.valid = authValue !== '';
setAuthValue('') is alreading doing this.